tests/Tests.AzureAppConfiguration/PushRefreshTests.cs

```diff
@@ -0,0 +1,334 @@
+                                    EventType = "eventType.KeyValueModified",
+                                    SyncToken = "SyncToken1"
+                                    },
+              new PushNotification  {
+                                    ResourceUri = new Uri("https://store2.resource.io/kv/searchQuery2"),
+                                    EventType = "eventType.KeyValueModified",
+                                    SyncToken = "SyncToken2"
+                                    },
+              new PushNotification  {
+                                    ResourceUri = new Uri("https://store2.resource.io/kv/searchQuery2"),
+                                    EventType = "eventType.KeyValueDeleted",
+                                    SyncToken = "SyncToken1"
+                                    },
+              new PushNotification  {
+                                    ResourceUri = new Uri("https://store2.resource.io/kv/searchQuery2"),
+                                    EventType = "eventType.KeyValueDeleted",
+                                    SyncToken = "SyncToken2"
+                                    }
+            };
+
+        List<PushNotification> invalidPushNotificationList = new List<PushNotification>
+            {
+              new PushNotification  {
+                                    ResourceUri = null,
+                                    EventType = "eventType.KeyValueModified",
+                                    SyncToken = "SyncToken1;sn=001"
+                                    },
+              new PushNotification  {
+                                    ResourceUri = new Uri("https://store2.resource.io/kv/searchQuery2"),
+									EventType = null,
+                                    SyncToken = "SyncToken2"
+                                    },
+              new PushNotification  {
+                                    ResourceUri = new Uri("https://store1.resource.io/kv/searchQuery1"),
+									EventType = "eventType.KeyValueDeleted",
+                                    SyncToken = null
+                                    },
+              new PushNotification  {
+                                    ResourceUri = null,
+                                    EventType = "eventType.KeyValueDeleted",
+                                    SyncToken = null
+                                    },
+              new PushNotification  {
+                                    ResourceUri = null,
+                                    EventType = null,
+                                    SyncToken = null
+                                    }
+            };
+
+		Dictionary<string, EventGridEvent> eventGridEventList = new Dictionary<string, EventGridEvent>
+		{
+            {
+                "sn;Vxujfidne",
+            new EventGridEvent(
+                "https://store1.resource.io/kv/searchQuery1",
+                "eventType.KeyValueModified", "2",
```

Comment:
>eventType.KeyValueModified [](http://example.com/codeflow?start=17&length=26)

Is this the actual EventType from EventGridEvent? According to our documentation, the event type should be:
`Microsoft.AppConfiguration.KeyValueModified` or `Microsoft.AppConfiguration.KeyValueDeleted`.

It doesn't matter what event type you use in this test because we dont even use it, but for accuracy and future-proofing the tests, lets use the real event types.

Reply:
Still need to address this.